src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -3583,6 +3583,7 @@ statfs(const char *pathname, struct statfs *sfs)
 	sfs->f_files  = -1;
 	sfs->f_ffree  = -1;
 	sfs->f_bavail = sfs->f_bfree;
+        sfs->f_fsid   = DAOS_SUPER_MAGIC;
```

Comment:
should this also happen in src/client/dfuse/ops/fsstat ?

Reply:
i wish!  but we covered that a few months ago (short version: boo fuse! )  https://daosio.atlassian.net/browse/DAOS-15304

Reply:
this is funny.. i pushed a similar PR:
https://github.com/daos-stack/daos/pull/15954

Reply:
im fine with this one instead. but this means you need to handle DAOS_SUPER_MAGIC in MPIIO too, right?

Reply:
it appears ROMIO is the only code in the world that uses DAOS_SUPER_MAGIC -- you invented/contributed that magic value back in the original DAOS driver commit 2dfc5ae38 .  ROMIO's definitely handling DAOS_SUPER_MAGIC already.

Reply:
you are right.. it's been a while ..